doc/release-notes-13152.md

```diff
@@ -3,3 +3,4 @@ New RPC methods
 
 - `getnodeaddresses` returns peer addresses known to this node. It may be used to connect to nodes over TCP without using the DNS seeds.
 - `listwalletdir` returns a list of wallets in the wallet directory which is configured with `-walletdir` parameter.
+- `deriveaddress` returns the address corresponding to an [output descriptor](/doc/descriptors.md).
```

Comment:
This doesn't seem to belong in this file (the file is specific per PR).

Reply:
listwalletdir was already hijacking that file, so I joined, but I can add a new one.

Reply:
In which case we could add them to the main file in the first place.

Reply:
This file was removed in master. Also, if another patch added a line in the meantime, git(hub) would also report a merge conflict.